### PR TITLE
Add HF application example

### PR DESCRIPTION
**Pull request by alexandredeleze**

### CODE REVIEW

File: hf-example/index.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+    <body>
+        <div class="container">
+            <h1>Pryv.io collect and visualize HF Data</h1>
+
+            <div class="card">
+                <div class="card-body">
+                    <h2 class="card-title">Welcome</h2>
+                    <br />
+                    <p>Click on fetch -> login -> mouse tracker</p>
```
Comment (princesseana) [inline hf-example/index.html:26]: add the story behind (I will do it)

File: hf-example/index.html [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+                </div>
+            </div>
+
+            <div class="card" id="service">
+                <div class="card-body">
+                    <button class="collapsible">
+            Service information
+          </button>
+                    <div class="service_info">
+                        <br> Enter the <a href="https://api.pryv.com/reference/#service-info">service information URL</a> to fetch the required information to customize your welcome message and fetch the URL for the authentication process.
```
Comment (princesseana) [inline hf-example/index.html:37]: Enter the <a href="https://api.pryv.com/reference/#service-info">service information URL</a>  for the authentication process.

File: hf-example/index.html [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+                        <span id="pryv-button"></span>
+                    </div>
+                </div>
+            </div>
+
+            <div class="card" id="mouse-tracker">
+                <div class="card-body">
+                    <h2 class="card-title">Mouse Tracker</h2>
+                    <br/>
+                    <p>You can select the image of your choice in the selector. If you change the image, it will erease your drawing on the screen. You need to left click to begin and to stop to draw.</p>
```
Comment (princesseana) [inline hf-example/index.html:72]: and to stop drawing.

File: hf-example/index.html [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+                    <br />
+                    <span id="frequency-mouse"></span>
+                    <br/>
+                    <div id="canvas-container">
+                        <canvas class="canvas" id="render-canvas" height="400px" width="600px"></canvas>
+                        <div id="image-button">
+                            <button id="previous" class="image-button">Previous</button>
+                            <button id="current" class="image-button">Current</button>
+                            <button id="next" class="image-button">Next</button>
+                        </div>
```
Comment (princesseana) [inline hf-example/index.html:101]: We need to add a link to a previous web app "Next example: View and Share Data" (https://github.com/pryv/app-web-examples/tree/master/view-and-share)

**Review by princesseana — COMMENTED** [review]
I am writing the "Welcome" and the readme according to the story. 

@alexandredeleze can you add a link to the "next" tutorial "View and share data" (https://github.com/pryv/app-web-examples/tree/master/view-and-share) at the buttom of the page ? (you can see the example here https://api.pryv.com/app-web-examples/onboarding/)

File: hf-example/index.html (hunk 2 above)
Comment (alexandredeleze) [reply hf-example/index.html]: Fixed

File: hf-example/index.html (hunk 3 above)
Comment (alexandredeleze) [reply hf-example/index.html]: Fixed

File: hf-example/index.html (hunk 4 above)
Comment (alexandredeleze) [reply hf-example/index.html:101]: Fixed

File: hf-example/index.html [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+                        <span id="pryv-button"></span>
+                    </div>
+                </div>
+            </div>
+
+            <div class="card" id="mouse-tracker">
+                <div class="card-body">
+                    <h2 class="card-title">Mouse Tracker</h2>
+                    <br/>
+                    <p>You can select the image of your choice in the selector. If you change the image, it will erease your drawing on the screen. You need to left click to begin and to stop drawing.</p>
```
Comment (princesseana) [inline hf-example/index.html:72]: erase

**Pull request merged**